packages/curve-ui-kit/src/themes/button/mui-button.ts

```diff
@@ -65,7 +66,20 @@ export const defineMuiButton = ({ Button, Text }: DesignSystem): Components['Mui
   return {
     styleOverrides: {
       root: {
-        variants: Object.entries(colors).map(([color, style]) => ({ props: { color }, style })),
+        variants: [
+          ...Object.entries(colors).map(([color, style]) => ({ props: { color }, style })),
+          {
+            props: {
+              variant: 'link',
+            },
+            style: {
+              textTransform: 'none',
+              padding: 0,
+              height: `${Sizing[400]} !important`,
```

Comment:
We should document the `!important` hack
```suggestion
              height: `${Sizing[400]} !important`, // unfortunately the sizes below are more specific
```